app.js

```diff
@@ -25,4 +26,5 @@ app.use(morgan("tiny"));
 app.use(express.static("public"));
 app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
-app.use("/", grocerieRouter);
+app.use("/list", listRouter);
+app.use("/", grocerieRouter);
```

Comment:
i don't like this route ("/") has not meaning. it should be something like "/items" or "/grocery"